notebooks/02_model/cornac_bpr_deep_dive.ipynb

```diff
@@ -0,0 +1,595 @@
+{
```

Comment:
<p>Now that our model is fitted </p><p>-&gt;</p><p>Now that our model is trained</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/microsoft/recommenders/pull/937/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='15'/>

Reply:
updated!